crates/polars-plan/src/dsl/python_udf.rs

```diff
@@ -17,14 +18,15 @@ use super::expr_dyn_fn::*;
 use crate::constants::MAP_LIST_NAME;
 use crate::prelude::*;
 
-// Will be overwritten on python polar start up.
+// Will be overwritten on Python Polars start up.
 pub static mut CALL_SERIES_UDF_PYTHON: Option<
     fn(s: Series, lambda: &PyObject) -> PolarsResult<Series>,
 > = None;
 pub static mut CALL_DF_UDF_PYTHON: Option<
     fn(s: DataFrame, lambda: &PyObject) -> PolarsResult<DataFrame>,
 > = None;
-pub(super) const MAGIC_BYTE_MARK: &[u8] = "POLARS_PYTHON_UDF".as_bytes();
+pub(super) const MAGIC_BYTE_MARK_UDF: &[u8] = "POLARS_PYTHON_UDF".as_bytes();
```

Comment:
I think we can keep one bytemark here. As you already know the `UDF` or `GET_OUTPUT` from the serde call. So `POLARS_PYTHON_UDF` would suffice, I think. Ideally it could be something shorter.

Reply:
> Ideally it could be something shorter.

I unified the two markers and changed it to `PLPYUDF` - saves a few bytes at least.